generators/chipyard/src/main/scala/upf/chisel/package.scala

```diff
@@ -0,0 +1,10 @@
+// See LICENSE for license details
+package chipyard.upf
+
+package object chisel {
```

Comment:
Why is this a `package object chisel`? Why not just call this `package chipyard.upf.chisel`?

Reply:
Also, I feel like this can be combined into the `UPFAspect.scala` file.